api/config.sample.js

```diff
@@ -85,9 +85,9 @@ var countlyConfig = {
     },
     /**
     * Default proxy settings, if provided then countly uses ip address from the right side of x-forwarded-for header ignoring list of provided proxy ip addresses
-    * @type {array=} 
+    * @type {array=}
     */
-    ignoreProxies: [/*"127.0.0.1"*/],
```

Comment:
I think here we still need IP address and not hostname, because we expect it in IP address header in nginx